modules/gapi/include/opencv2/gapi/video.hpp

```diff
@@ -248,6 +295,45 @@ The operation generates a foreground mask.
 */
 GAPI_EXPORTS GMat BackgroundSubtractor(const GMat& src, const cv::gapi::video::BackgroundSubtractorParams& bsParams);
 
+/** @brief Standard Kalman filter algorithm <http://en.wikipedia.org/wiki/Kalman_filter>.
+
+@return Output image is predicted or corrected state. They can be 32-bit or 64-bit float
```

Comment:
> image

or matrix?

---

Huge `@return` section doesn't look good in [documentation](http://pullrequest.opencv.org/buildbot/export/pr/18869/docs/d4/d55/group__gapi__video.html#ga01d36765f6c877f577f15f04850ffa19).

Separate details from `@return` / `@param` sections.
Return section should have just short meaning and expected type:
- `@return Output image is predicted or corrected state. Type is @ref CV_32F or @ref CV_64F`

---

Prefer to place `@return` section after all including `@params`

Reply:
Done.